jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedPanel.kt

```diff
@@ -245,15 +246,21 @@ class GettingStartedPanel(private val project: Project) : BorderLayoutPanel() {
                             label(message("codewhisperer.gettingstarted.panel.licence_comment")).applyToComponent { foreground = PanelConstants.TEXT_FONTCOLOR }
                         }
                         row {
-                            browserLink(message("aws.onboarding.getstarted.panel.login_with_iam"), url = CODEWHISPERER_LEARN_MORE_URI)
+                            text(message("aws.onboarding.getstarted.panel.login_with_iam")) {
+                                try {
+                                    SetupAuthenticationDialog(project).show()
+                                } catch (e: Exception) {
+                                    throw Exception("Unable to pop up the IAM Identity Center Authentication Dialog")
+                                }
```

Comment:
?

Reply:
Using a catch block helps us track errors and in a recent situation while I am testing locally, it didn't pop up when I clicked a link, making it useful to have an exception message.